iron-autogrow-textarea.html

```diff
@@ -49,8 +41,12 @@
       word-wrap: break-word;
     }
 
-    ::content textarea {
+    textarea {
+      position: relative;
```

Comment:
Hmm, this unfortunately means that the `iron-autogrow-textarea` is completely unstyled (unlike a normal `textarea`). Something something visual opinions.

Can we remove the outline/border, and add them in the `paper-input-area` where they are needed?